projects/ngx-wig/src/lib/ngx-wig-component.css

```diff
@@ -300,12 +300,12 @@ div.select-menu>select {
     border-radius: 3px 3px 0 0;
 }
 
-.dropdown {
+.nwmenu-dropdown {
```

Comment:
Can we say `.nw-menu-dropdown`?

Reply:
See [my comment](https://github.com/stevermeister/ngx-wig/pull/180#discussion_r989708111) above.